File: PRAK405-2310817210021-RAKHMAD FAUZHAN RAMADHAN.py
def main():
    print("Input")
    n, k = map(int, input().split())

    total = 0
    for i in range(1, n + 1):
        barisan = 0
        for j in range(i, 0, -1):
            nilai = j * k
            barisan += nilai
            print(f"({j} * {k}) = {nilai}", end=" + " if j > 1 else "\n")

        total += barisan

    print("\nOutput")
    print(total)

if __name__ == "__main__":
    main()

File: MODUL 4/PRAK405-2310817210021-RAKHMAD FAUZHAN RAMADHAN.c
#include <stdio.h>

int main() {
int n, k;
printf("Input\n");
scanf("%d %d", &n, &k);

int total = 0;
for (int i = 1; i <= n; i++) {
int barisan = 0;
for (int j = i; j >= 1; j--) {
int nilai = j * k;
barisan += nilai;
printf("(%d * %d) = %d", j, k, nilai);

if (j > 1) {
printf(" + ");
} else {
printf("\n");
}
}
total += barisan;
}

printf("\nOutput\n");
printf("%d\n", total);

return 0;
}
